source/_integrations/ecoforest.markdown

```diff
@@ -14,28 +14,14 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Ecoforest integration allows monitoring and control of local [Ecoforest](https://ecoforest.com) fireplaces in Home Assistant. These fireplaces are manufactured in Spain.
+The Ecoforest integration allows monitoring and control of local [Ecoforest](https://ecoforest.com) fireplaces in Home Assistant.
 
 There is currently support for the following device platforms within Home Assistant:
 
 - [Sensor](#sensor)
 
 {% include integrations/config_flow.md %}
 
-<div class='note'>
-Devices must first be connected to Wi-Fi or LAN using the app or website provided by the device manufacturer. It is not possible to connect devices to Wi-Fi using this integration.
-</div>
-
-### Credentials
-
-To configure the Ecoforest integration you will need to enter your Ecoforest credentials which are the same ones you would use with the manufacturer app. The image below shows how to obtain the credentials:
-
-- Username: use the serial number of the device as identified by 1.
-- Password: use the first 8 characters of the wifi password as identified by 4.
-
-![Ecoforest Credentials](/images/integrations/ecoforest/credentials.png)
```

Comment:
Why has this been removed?

Reply:
> @joostlek @pjanuario It doesn't make much sense to have install instructions above how to get the details needed to install. This is a prerequisite...

This was my understanding from previous comment in the other PR, maybe I misunderstood. So what do you mean exactly?

Reply:
I mean this should have been listed before/above the config flow procedure (and the paragraph to be renamed to "Prerequisites" to match our common standards).

../Frenck